src/app/login-form/login-form.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
```

Comment:
This file has no spec file

You can generate services with angular cli as well

```sh
ng g s login/login
```